UnitsNet.Tests/CustomCode/VolumeTests.cs

```diff
@@ -75,6 +77,8 @@ public class VolumeTests : VolumeTestsBase
 
         protected override double TeaspoonsInOneCubicMeter => 202884.13621105801;
 
+        protected override double TotesInOneCubicMeter => 0.9606256824;
```

Comment:
I don't think I consider tote to be a unit of measurement. It is a physical container that has some common sizes widely used, but this definition belongs in an application, not in Units.NET.

Source, wiki:
> IBCs can be manufactured to a customer's exact requirements in terms of capacity, dimensions, and material. The length and width of an IBC are usually dependent on the pallet dimension standard of a given country

and

> The most common sizes are 1,040 liters or 275 U.S. gallons or 229 imperial gallons and 1,250 liters or 330 U.S. gallons or 275 imperial gallons (the 1040 liter IBCs are often listed as being 1000 liters)

https://en.wikipedia.org/wiki/Intermediate_bulk_container

The same reasoning on drums.
If you agree, please remove these two definitions from the PR.